packages/bundler/src/BundlerCollectorTracer.ts

```diff
@@ -159,28 +175,47 @@ export function bundlerCollectorTracer (): BundlerCollectorTracer {
         }
       }
 
+      if (log.getDepth() === 1) {
+        if (opcode === 'CALL' || opcode === 'STATICCALL') {
+          // stack.peek(0) - gas
+          const addr = toAddress(log.stack.peek(1).toString(16))
+          const topLevelTargetAddress = toHex(addr)
+          // stack.peek(2) - value
+          const ofs = parseInt(log.stack.peek(3).toString())
+          let len = parseInt(log.stack.peek(4).toString())
+          if (len > 4) len = 4
```

Comment:
How can a top-level call from the EntryPoint be `< 4` bytes ? I don't think  peeking at `len` is even needed here.

Reply:
yes, entrypoint doesn't make calls without methosig.
this check just for completeness, as the code tries to be generic, and not only for entrypoint.